create-tables.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
+
+  CREATE TABLE IF NOT EXISTS public.projects (
+    "name" varchar(100) NOT NULL DEFAULT ''::character varying,
+    version_id int4 NOT NULL DEFAULT 0,
+    last_accessed timestamptz NULL,
+    CONSTRAINT projects_pkey PRIMARY KEY (name)
+  );
+  CREATE INDEX IF NOT EXISTS projects_index_last_accessed
+    ON public.projects
+    USING btree
+    (last_accessed);
+
+  CREATE TABLE IF NOT EXISTS public.url_index_store (
+    project_name varchar(100) NOT NULL DEFAULT ''::character varying,
+    url varchar(2000) NOT NULL,
+    "path" varchar(2000) NOT NULL,
+    CONSTRAINT url_index_store_pkey PRIMARY KEY (project_name, url)
```

Comment:
I thought that we were going to change the primary key to `(project_name, path)` to resolve the `ON CONFLICT` issue?

Reply:
Actually, we do a query like `select path from url_index_store where project_name = x and url = y;` (at `getPathForURLInProject()`. The logic is basically "given a project and url, have we ever previously seen this at a path?"

Reply:
We could chat through it after standup though :+1:

Reply:
Discussed this in standup, this is good :+1: